suite-native/module-onboarding/src/screens/FeatureReceiveScreen.tsx

```diff
@@ -1,17 +1,23 @@
 import { useNavigation } from '@react-navigation/native';
 
+import { useIsUsbDeviceConnectFeatureEnabled } from '@suite-native/feature-flags';
 import {
     OnboardingStackParamList,
     OnboardingStackRoutes,
     StackNavigationProps,
 } from '@suite-native/navigation';
 import { Box } from '@suite-native/atoms';
+import { useTranslate } from '@suite-native/intl';
 
 import { OnboardingFooter } from '../components/OnboardingFooter';
 import { OnboardingScreen } from '../components/OnboardingScreen';
 import { CoinsSvg } from '../components/CoinsSvg';
 
 export const FeatureReceiveScreen = () => {
```

Comment:
I'd separate the components outside of the "wrapper". No need to have that encapsulated and is cheaper when FF changes I think

Reply:
done in 829658806